### PR TITLE
Feature/lol/improve bar chart

### PR DESCRIPTION
**Pull request by lliss**
Connects #410 
To test, re-bundle and start a model. On the modelling page, look at the bar chart and resize the window. You should not get bad overlaps unless the window is very small. Legend should be in proper order.

### CODE REVIEW

File: src/mmw/js/src/core/chart.js [hunk 1]
```diff
@@ -46,6 +46,16 @@ function makeBarChart(selector, data, indVar, depVars, options) {
         bars,
         legend;
 
+    /**
+     * Drawing constants. AXIS_OFFSET provides a forced px value offset to
+     * prevent the charts from sitting on top of the axis labels.
+     * CHART_CONTINER_PERCENTAGE sets the actual bar chart area width to a
+     * percentage of the overall svg width. This helps ensure the chart doesn't
+     * take over the whole container, leaveing room for the legend.
```
Comment (caseypt) [inline src/mmw/js/src/core/chart.js]: typo: leaveing > leaving

File: src/mmw/js/src/core/chart.js [hunk 2]
```diff
@@ -46,6 +46,16 @@ function makeBarChart(selector, data, indVar, depVars, options) {
...
+     */
+    var AXIS_OFFSET = 20,
+        CHART_CONTINER_PERCENTAGE = 0.8;
```
Comment (caseypt) [inline src/mmw/js/src/core/chart.js]: typo: CONTINER > CONTAINER

**Comment by lliss** [comment]
Rebased.

**Comment by lliss** [comment]
Rebased again on top of develop.

**Comment by lliss** [comment]
@azaveaci rebuild

**Comment by caseypt** [comment]
+1 Looks good. 

This is not caused by this PR, but the `Level` text becomes blurry at the `1279px`, probably because of a media query. If you are familiar with how to fix this because of working on this stuff, consider fixing it.

1280
selection_013

1279
selection_012

**Comment by lliss** [comment]
Moved that to a new issue https://github.com/WikiWatershed/model-my-watershed/issues/444

**Pull request merged**